src/lib/caching/stores/GuildEmojiStore.ts

```diff
@@ -37,7 +38,11 @@ export class GuildEmojiStore extends DataStore<GuildEmoji> {
 	 * @param requestOptions The additional request options.
 	 * @see https://discord.com/developers/docs/resources/emoji#create-guild-emoji
 	 */
-	public async add(data: GuildEmojiStoreAddData, requestOptions: RequestOptions = {}): Promise<GuildEmoji> {
+	public async add({ image, ...options }: GuildEmojiStoreAddData, requestOptions: RequestOptions = {}): Promise<GuildEmoji> {
+		const data: GuildEmojiStoreAddData = {
+			image: image ? await resolveImageToBase64(image) : image,
```

Comment:
Wh..Image is REQUIRED for creating an emoji. This entire ternary is confusing as a whole

Reply:
both image and name right?

Reply:
Yes

Reply:
Yes, fixed in 6005cd9.